meta-facebook/yv4-sd/src/platform/plat_def.h

```diff
@@ -70,6 +70,7 @@
 #define HOST_KCS_PORT kcs3
 #define BMC_USB_PORT "CDC_ACM_0"
 #define MCTP_I3C_PEC_ENABLE 1
+#define PLAT_PLDM_MSG_TIMEOUT_MS 30000
```

Comment:
@SaraLin-wiwynn why not just change PLDM_MSG_TIMEOUT_MS to 30s? Why do we need platform specialization for this?

Reply:
Hi Amithash,
We don't want to modify the settings of other projects.
Afraid of affecting the functions of other projects, thus we use the definition of platform level.